CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # [Versions](https://mui.com/versions/)
 
+## 7.0.0-beta.5
+
+<!-- generated comparing v7.0.0-beta.3..master -->
+
+_Mar 18, 2025_
+
+A big thanks to the 10 contributors who made this release possible.
+
+### `@mui/material@7.0.0-beta.5`
+
+- [Accordion] Add missing `root` slot (#45532) @sai6855
+- [AccordionSummary] Add slots and slotProps (#45559) @sai6855
+- [ListItemText] Add missing `root` slot (#45540) @sai6855
+- [SpeedDial] Add missing `root` slot (#45541) @sai6855
+- [StepLabel] Add missing root slot (#45603) @sai6855
+- [Switch] Add slots and slotProps (#45595) @siriwatknp
+- [Tooltip] Allow auto placement on tooltip (#45399) @Jtaks
+- [TextareaAutosize] Fix ResizeObserver causing infinite `selectionchange` loop (#45351) @mj12albert
+- [useScrollTrigger] Do nothing if target is null (#45441) @vipierozan99
+- [core] Rename some `@mui/material/styles/createTypography` exports (#45558) @Janpot
```

Comment:
Just removing `@mui/material/styles/createTypography` as it's not something we publicly export.
```suggestion
- [core] Rename some `./styles/createTypography` exports (#45558) @Janpot
```